packages/tools/devtools/devtools/src/index.ts

```diff
@@ -27,10 +27,12 @@ import {
 	type HasContainerKey,
 	type IFluidDevtools as IDevtoolsBase,
 	type IDevtoolsLogger,
+	type IFluidDevtools,
 	initializeDevtools as initializeDevtoolsBase,
 } from "@fluidframework/devtools-core/internal";
 import type { IFluidContainer } from "@fluidframework/fluid-static";
 import { isInternalFluidContainer } from "@fluidframework/fluid-static/internal";
+import { FluidDevtools } from "../../devtools-core/lib/FluidDevtools.js";
```

Comment:
We can't import from another package this way. Sorry, I noticed this after approving.

Reply:
Note, if you move the below functionality into `devtools-core` (see my other comments), this is moot. But in general, importing like this is not supported.

Reply:
Moved this function to `devtools-core`